flight/Modules/Attitude/smallf1/attitude.c

```diff
@@ -73,10 +72,17 @@ static SensorSettingsData sensorSettings;
 static void AttitudeTask(void *parameters);
 
 static float gyro_correct_int[3] = {0,0,0};
+
+#ifdef COPTERCONTROL
```

Comment:
We're using both.  SMALLF1 means we're on a resource limited platform.  COPTERCONTROL is used within the smallf1 attitude module to indicate that we want the few lines particular to CC/CC3D that don't apply to Naze32.